patches/chrome-android-BUILD.gn.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/chrome/android/BUILD.gn b/chrome/android/BUILD.gn
+index 2423d123e6b32960ef450f101ce1b79c3fd418df..d30382da1d6a9bf9e5e2c19edaeb9deed3c60205 100644
+--- a/chrome/android/BUILD.gn
++++ b/chrome/android/BUILD.gn
+@@ -38,7 +38,10 @@ if (enable_vr) {
+   import("//chrome/android/features/vr/vr_module_tmpl.gni")
+ }
+ 
+-manifest_package = "org.chromium.chrome"
++declare_args() {
++  manifest_package = "org.chromium.chrome"
```

Comment:
please either leave out the indent or do the whole thing in one line to reduce patch size